app/controllers/api/v1/users_controller.rb

```diff
@@ -1,20 +1,10 @@
 class Api::V1::UsersController < ApplicationController
-  def index
-    render json: UserSerializer.new(User.all)
-  end
-
+  before_action :authorize_access_request!
+  
   def show
     render json: UserSerializer.new(User.find(params[:id]))
   end
 
-  def create
```

Comment:
The sessions controller made this action redundant so I removed it for the time being